web/src/main/java/org/springframework/security/web/authentication/DelegatingAuthenticationConverter.java

```diff
@@ -38,12 +37,11 @@ public final class DelegatingAuthenticationConverter implements AuthenticationCo
 
 	public DelegatingAuthenticationConverter(List<AuthenticationConverter> delegates) {
 		Assert.notEmpty(delegates, "delegates cannot be null");
-		this.delegates = new ArrayList<>(delegates);
+		this.delegates = delegates;
```

Comment:
Hi @kse-music , delegates passed to the constructor must be wrapped in a new list, only then can we suggest that they not be changed from the outside. Your solution is not safe in this regard, so I suggest leaving it as it is.